### ISSUE TITLE
hcaptcha

### ISSUE THREAD

**Issue by yhlalex** [justinchuntingho/LIHKGr#11]
Is this still working? I think they migrated from recaptcha to hcaptcha and when I was running your code, it got stuck on the hcaptcha page.

**Comment by justinchuntingho** [comment]
You are correct. I am still figuring a way to bypass it (PR is highly welcomed).

**Comment by darkjackx** [comment]
i am making a bot using python + selenium for lihkg
this is the way to bypass on selenium
option.add_argument("--disable-blink-features=AutomationControlled")
